docs/getting_started/ibexa_cloud_guide.md

```diff
@@ -0,0 +1,111 @@
+---
+description: The Ibexa Cloud product guide provides a full description of its features as well as the benefits it brings to the client.
+---
+
+# [[= product_name_cloud =]] product guide
+
+## What is [[= product_name_cloud =]]
+
+[[= product_name_cloud =]] is a cloud hosting platform that enables you to host your application in the cloud by using the [Platform.sh](https://platform.sh/) service.
+It also establishes the framework for the potential growth and improves project delivery.
+As a diverse Platform as a Service (PaaS), it's intended to let you to concentrate on the important things.
+
+[[= product_name_cloud =]] is a part of [[= product_name =]] - a software that is designed to provide your business with all the features, functionality, and support you need to transform your business for the digital age.
+
+![Ibexa Cloud - part of Ibexa DXP](ibexa_cloud_dxp.png)
+
+## Availability
+
+[[= product_name_cloud =]] is available in all [[= product_name =]] editions.
+
+## How does [[= product_name_cloud =]] work
+
+Ibexa's Cloud Hosting Platform is designed to streamline the development and testing processes, allowing you to deliver new features much faster.
+It ensures that developers spend less time managing databases, queues, search engines, and operating systems and more time to important development activities.
```

Comment:
Sentence is updated now: It ensures that developers dedicate more time to important development activities rather than maintaining databases, queues, search engines, and operating systems.